.github/workflows/prettier.yml

```diff
@@ -0,0 +1,18 @@
+name: Prettier
```

Comment:
Happy to see Prettier added!

Reply:
In addition to being a Typescript fan boy I am _also_ a Prettier fanboy 😀